.travis.yml

```diff
@@ -23,10 +23,6 @@ jobs:
       env: PURPOSE='Headers'
       script: ./scripts/ci/precheck_header.sh
       python: 3.6
-    - stage: precheck
```

Comment:
I will revert this and also remove the privates after the azure-mgmt-compue gets published